social-network/src/MissedArticleData.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import MissedArticles from './missed-articles.json';
+import PropTypes from 'prop-types';
+import Image from 'react-bootstrap/Image'
+import Media from 'react-bootstrap/Media'
+import './ArticleData.css';
+
+
+function MissedArticleData({ index }) {
+  return (
+    <div className='missedArticleWrapper'>
+      <Image src={MissedArticles[index]['image']} className=' missedArticleImg' />
```

Comment:
Instead of reaching back into the articles array, you can also pass the entire article as a prop so it's a little easier to work with.